test/integration/targets/iam_user/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+- name: set up aws connection info
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token }}"
```

Comment:
please use `security_token | default(omit)` some folks (myself included) run the tests using just an access/secret key pair rather than using temporary tokens.